assets/prototype/domain/eventEditor/data/queries/tickets/test/useTicketPrices.test.ts

```diff
@@ -0,0 +1,86 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import useTicketPrices from '../useTicketPrices';
+import { ApolloMockedProvider } from '../../../../context';
+import { nodes } from './data';
+import useInitTicketTestCache from './useInitTicketTestCache';
+import useInitPriceTestCache from '../../prices/test/useInitPriceTestCache';
+import { useRelations } from '../../../../../../application/services/apollo/relations';
+
+describe('useTicketPrices()', () => {
+	const wrapper = ApolloMockedProvider();
+	const existingTicket = nodes[0];
+
+	it('returns empty array for ticket prices when the ticket exists and the cache is empty', () => {
+		const { result } = renderHook(() => useTicketPrices(existingTicket.id), { wrapper });
+
+		const { current: cachedTicketPrices } = result;
```

Comment:
Amazing, thx for adding a bit of context :+1: 
`expect(result.current).toEqual([]);` wouldn't be clear enough

Reply:
Just by reading this line in isolation `expect(cachedTicketPrices).toEqual([]);`
it makes it easy to understand what is the result of the hook